packages/terra-date-time-picker/src/_TimeClarification.jsx

```diff
@@ -25,13 +25,33 @@ const propTypes = {
    * Callback function indicating the after time change option was selected.
    */
   onStandardTimeButtonClick: PropTypes.func.isRequired,
+  /**
+   * Callback function indicating the DST offset button was selected.
+   */
+  onOffsetButtonClick: PropTypes.func.isRequired,
+  /**
+   * Callback function indicating the modal is requesting to close.
+   */
+  onRequestClose: PropTypes.func.isRequired,
+  /**
+   * A callback function to let the containing component (e.g. modal) to regain focus.
+   */
+  releaseFocus: PropTypes.func,
+  /**
+   * A callback function to request focus from the containing component (e.g. modal).
+   */
+  requestFocus: PropTypes.func,
 };
 
 const defaultProps = {
   isOpen: false,
   isOffsetButtonHidden: false,
   onDaylightSavingButtonClick: undefined,
   onStandardTimeButtonClick: undefined,
+  onOffsetButtonClick: undefined,
```

Comment:
I didn't see any Jest tests updated for these new props, which I find a little surprising. Is there a reason they are not needed?

Reply:
There wasn't previously a jest test for it. I'll add one.